packages/terra-clinical-item-display/src/ItemDisplay.jsx

```diff
@@ -82,6 +83,11 @@ const ItemDisplay = ({
     { 'strike-through': textStyle === TextStyles.STRIKETHROUGH },
   ]);
 
+  // TODO: remove this warning in the next major release.
+  if (isDisabled) {
```

Comment:
I see it's just a warning, but should we disable the no-console linter rule for this line?

Reply:
Disabled here: 51180274acf452a3d57b7018e52ed217271efdfe